python/tvm/relay/frontend/tensorflow.py

```diff
@@ -614,6 +614,53 @@ def _impl(inputs, attr, params, mod):
         return out
     return _impl
 
+def _nms():
+    def _impl(inputs, attr, params, mod):
+        # Get parameter values
+        max_output_size = int(np.atleast_1d(inputs[2].data.asnumpy().astype("int64"))[0])
```

Comment:
Handle symbolic max_output_size:
```suggestion
        try:
            max_output_size = int(np.atleast_1d(inputs[2].data.asnumpy().astype("int64"))[0])
        except Exception:
            try:
                max_output_size = _infer_value(inputs[2], params, mod).asnumpy().astype("int64").tolist()[0]
            except Exception:
                max_output_size = inputs[2]
```